### PR TITLE
Add Pkg package and compat for it to packages

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

**Comment by vchuravy** [comment]
Ah this is missing the "Pkg package" part.

File: A/AbInitioSoftwareBase/Compat.toml [hunk 1]
```diff
@@ -6,6 +6,7 @@ julia = "1"
 
 ["0-0.2.0"]
 Configurations = "0.3-0.15"
+Pkg = "1"
```
Comment (GunnarFarneback) [inline A/AbInitioSoftwareBase/Compat.toml:10]: I don't know exactly what changes this is supposed to make, but only updating compat for the oldest version seems unreasonable.

Comment (GunnarFarneback) [reply A/AbInitioSoftwareBase/Compat.toml:10]: Ok thanks. That was the connection I was missing.

**Comment by KristofferC** [comment]
I think LazyArtifacts have to be made upgradable first.

**Comment by vchuravy** [comment]
Yeah maybe it's time to redo #92622

**Pull request closed without merging**